transport/src/androidUnitTest/kotlin/transport/core/messagingclient/MCAuthTests.kt

```diff
@@ -3,6 +3,7 @@ package transport.core.messagingclient
 import assertk.assertThat
 import assertk.assertions.isEqualTo
 import com.genesys.cloud.messenger.transport.auth.NO_JWT
+import com.genesys.cloud.messenger.transport.auth.NO_REFRESH_TOKEN
```

Comment:
General comment. It seems to me that feature branch is outdated. Recent copy of develop use different name for MCAuthTest (MessagingClientTest) I recommend to sync develop and feature branch now to avoid merging conflicts in the future.